webviz_subsurface/_models/parameter_model.py

```diff
@@ -106,11 +107,17 @@ def _check_if_sensitivity_run(self) -> bool:
         if "SENSNAME" not in self._dataframe:
             return False
 
-        if self.dataframe["SENSNAME"].isnull().values.any():
+        gen_kw_mask = self._dataframe["SENSNAME"].isnull()
+        if gen_kw_mask.values.any() and not allow_ensemble_type_mix:
             raise ValueError(
                 "Ensembles with and without sensitivity data mixed - this is not supported!"
             )
```

Comment:
Should this error message be updated slightly now? E.g. replace "this is not supported" with "this is not enabled".